compiler_opt/rl/inlining/__init__.py

```diff
@@ -14,11 +14,13 @@
 # limitations under the License.
 """Implementation of the 'inlining for size' problem."""
 
+from typing import Tuple, List
 import gin
 
 from compiler_opt.rl import problem_configuration
 from compiler_opt.rl.inlining import config
 from compiler_opt.rl.inlining import inlining_runner
+from compiler_opt.rl.inlining import inlining_spec
```

Comment:
spec->module_spec everywhere cause spec is too general so it can cause confusion